src/main/java/frc/team2412/robot/Subsystems.java

```diff
@@ -18,11 +18,11 @@
 public class Subsystems {
 	public static class SubsystemConstants {
 		public static final boolean DRIVEBASE_ENABLED = true;
-		public static final boolean ARM_ENABLED = true;
-		public static final boolean INTAKE_ENABLED = true;
+		public static final boolean ARM_ENABLED = false;
+		public static final boolean INTAKE_ENABLED = false;
```

Comment:
I'd recommend setting these back to `true` so we don't run into the same issue we had today.